item-descriptions.lua

```diff
@@ -41,7 +41,7 @@ the latter script, view-item-info will fall back to the former.
 ]]
 
 -- see http://dwarffortresswiki.org/index.php/cv:Item_token
----luacheck: global
+--luacheck: global
```

Comment:
This should be on the line immediately before the variable. It won't silence the warning if there's a blank line in between.

Reply:
Fixed... I think.